addons/blocks2image/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Save blocks as image",
-  "description": "Right click the editor workspace to export blocks as SVG/PNG images.",
+  "name": "Block images",
+  "description": "Right click the code area to save blocks as SVG/PNG images.",
```

Comment:
"Export" is fine, used in vanilla.

Reply:
I changed the description to include the keyword I removed from the title, but I'll revert this change (and the one above).